pkg/controllers/core/deployer_controller.go

```diff
@@ -103,11 +101,39 @@ func (r *DeployerReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	return result, err
 }
 
+func (r *DeployerReconciler) getAndTrackConfigMap(ctx context.Context, log logr.Logger, deployer *corev1alpha1.Deployer) (*corev1.ConfigMap, error) {
+	cm := &corev1.ConfigMap{}
+
+	versionKinds, _, err := r.Scheme.ObjectKinds(cm)
```

Comment:
we only need to lookup the kind for a generic Object, since we know this is a ConfigMap the kind will always be `ConfigMap` and the apiVersion will always be `v1`.

Reply:
I've reverted this back to the version before with the explicit version and kind.